.github/workflows/nl-action.yml

```diff
@@ -18,12 +18,14 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: 3.7
+    - uses: actions/cache@v2
```

Comment:
Great, with this cache we are able to reduce dependence installation time from 3m 50sec --> 56sec. Hence, reducing overall build run time.

Reply:
Nice! Works as expected then.